src/Blorc.OpenIdConnect/Services/Interfaces/IUserManager.cs

```diff
@@ -19,5 +16,9 @@ public interface IUserManager
         Task<TUser> GetUserAsync<TUser>(Task<AuthenticationState> authenticationStateTask, JsonSerializerOptions options = null);
 
         Task<TUser> GetUserAsync<TUser>(bool reload = true, JsonSerializerOptions options = null);
+
+        event EventHandler<UserInactivityEventArgs> UserInactivity;
+
+        event EventHandler UserActivity;
```

Comment:
EventHandler<UserActivityEventArgs> just in case we want something in the future about this. For example, to reset the counter, it would be good to have the remaining time, even on user activity (e.g. to reset the local timer).